comment/tests/test_utils.py

```diff
@@ -81,3 +82,38 @@ def test_get_comment_context_data(self):
         comment_context_data = get_comment_context_data(request)
         self.assertEqual(comment_context_data['comments'].paginator.per_page, 2)
         self.assertTrue(comment_context_data['comments'].has_next())
+
+
+class UtilsTest(TestCase):
+    """Test general purpose utilities that aren't necessarily related to a comment"""
+
+    def setUp(self):
+        self.len_id = 6
+
+    def test_id_generator_length(self):
+        self.assertEqual(self.len_id, len(id_generator()))
+
+    def test_id_generator_generates_different_ids(self):
+        self.assertNotEqual(id_generator(), id_generator())
+
+    def test_id_generator_prefix(self):
+        prefix = 'comment'
+        output = id_generator(prefix=prefix)
+        self.assertEqual(True, output.startswith(prefix))
```

Comment:
I see that you have removed most `self.assertEqual(True, condition)` with `self.assertTrue(condition)`. Both do the same work, in general `self.assertEqual` gives more verbose message on failing.

Reply:
you can use `assertEqual` for almost any case, but I would like to be specific when asserting booleans with `assertTrue` and `assertFalse`. I don't think that we need the verbose message here since we expect 2 values only `True` or `False`.

Reply:
It is more helpful when debugging in case the test fails.